blackboxopt/optimizers/testing.py

```diff
@@ -27,7 +27,7 @@ def _initialize_optimizer(
     seed=42,
 ) -> Optimizer:
     if space is None:
-        space = ps.ParameterSpace()
+        space = ps.ParameterSpace(seed=seed)
```

Comment:
This should not be necessary, since the `Optimizer` base class [seeds its search space](https://github.com/boschresearch/blackboxopt/blob/main/blackboxopt/base.py#L148-L149) when a seed is given. Was there a specific reason (i.e. something failing) that motivated you to change this?

Reply:
Nope, just didn't know if/where the space was seeded...

Reply:
Ok, then let's not seed it here in order to spot when optimizers don't call the super init properly.

Reply:
Done